elasticsearch-model/test/unit/adapter_mongoid_test.rb

```diff
@@ -75,13 +75,57 @@ def ids
     end
 
     context "Importing" do
-      should "implement the __find_in_batches method" do
-        DummyClassForMongoid.expects(:all).returns([])
-
+      setup do
         DummyClassForMongoid.__send__ :extend, Elasticsearch::Model::Adapter::Mongoid::Importing
+      end
+
+      should "raise an exception when passing an invalid scope" do
```

Comment:
Can you please move this after the "happy paths", ie. after line 128 in your patch?